content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -776,7 +777,7 @@ node/mynode drained
 
```

Comment:
Update `An example set of headers:` above with the uid header.

Reply:
Resolved, I just used a random hex string, but do you think I should do something that's more obviously a placeholder value (like `aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa` or `xxxxxxxx`)?

Reply:
I think it's fine as-is. You could switch to something that's a valid UUID but still obvious, eg `123e4567-e89b-42d3-a456-​556642440000`

Reply:
Ok, if it's fine with you I think I'll just leave it as is.